packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx

```diff
@@ -120,7 +125,7 @@ class CollapsibleMenuViewItemGroup extends React.Component {
     ]);
 
     return (
-      <ButtonGroup {...customProps} isMultiSelect={isMultiSelect} onChange={onChange} className={buttonGroupClassNames} selectedKeys={selectedKeys} aria-label={ariaLabel}>
```

Comment:
why did we removed ariaLabel ..? it is first class prop so we can still use them here correct..??

Reply:
Still consumer can able to pass aria-label as custom props in collapsible-menu that's why i have removed first class props, if required i can include first class props

Reply:
Updated - https://github.com/cerner/terra-framework/pull/1841/commits/f8e50c838fae764016564bd374289997cb145cba